Ladybird/default.nix

```diff
@@ -1,12 +1,7 @@
-{ pkgs ? import <nixpkgs> { } }:
-pkgs.mkShell.override
-{
-  stdenv = pkgs.gcc13Stdenv;
-}
-{
-  name = "ladybird";
+{ pkgs ? import <nixpkgs> { } }: with pkgs;
 
-  nativeBuildInputs = with pkgs; [
+mkShell.override { stdenv = gcc13Stdenv; } {
+  packages = [
```

Comment:
I think `qt6.qtsvg` can be removed from these dependencies (the Qt chrome has not needed that for a while).